src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -11,7 +13,8 @@
 (def Extensions
   "Schema for representing SSL Extensions. Maps from a keyword shortname to a
   string value."
-  {schema/Keyword schema/Str})
+  {schema/Keyword schema/Str
+   (schema/optional-key :subjectAlternativeName) [schema/Str]})
```

Comment:
1) agreed that subject_alt_name is way better; renamed
2) that schema is for what comes out of a cert for subjectAlternativeName, which is always going to be a list of strings. The ACEConfig schema is for the configuration side and remains unchanged.